src/ExceptionTransformer.ts

```diff
@@ -46,9 +53,78 @@ class ExceptionTransformer {
         exception.fallback_message
       );
     }
-
     return exceptionMap;
   }
+
+  public generateSpesificFieldError(errorInfo: Exception | null | undefined) {
```

Comment:
Can you explain in a comment line with a couple of words what actually this method does?

I think we should do more strict type checking 👇 
```suggestion
  public generateSpesificFieldError(
    errorInfo: Exception | null | undefined
  ): (fieldName: string) => () => string | null {
    const errorDetail = getErrorDetail(errorInfo);

    return function getError(fieldName: string): () => string | null {
      return errorDetail ? errorDetail[fieldName] : null;
    };
  }
```

Also, you can create a generic interface for return functions.

```suggestion
  public generateSpesificFieldError(
    errorInfo: Exception | null | undefined
  ): (fieldName: string) => ReturnFunction<string | null> {
    const errorDetail = getErrorDetail(errorInfo);

    return function getError(fieldName: string): ReturnFunction<string | null> {
      return errorDetail ? errorDetail[fieldName] : null;
    };
  }
```

```javascript
interface ReturnFunction<ValueType> {
  (): ValueType;
}
```